# HG changeset patch
# User Ray Donnelly <mingw.android@gmail.com>
# Date 1471185471 -3600
#      Sun Aug 14 15:37:51 2016 +0100
# Node ID 1792d96eaee85143f4426bbf5a8c21ac0f9b3bc2
# Parent  96f4dd23279f67691b74652b1ef59898dc8d9fdb
Turn off some pedantic warnings (unused variables and functions)

diff --git a/setup.py b/setup.py
--- a/setup.py
+++ b/setup.py
@@ -225,6 +225,9 @@
         define_macros.append(('RIF_HAS_RSIGHAND', 1))
         define_macros.append(('CSTACK_DEFNS', 1))
         define_macros.append(('HAS_READLINE', 1))
+    extra_compile_args.append('-Wno-unused-variable')
+    extra_compile_args.append('-Wno-unused-but-set-variable')
+    extra_compile_args.append('-Wno-unused-function')
 
 
     if sys.byteorder == 'big':
